src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -92,20 +89,37 @@ public async Task<IActionResult> Login(LoginViewModel viewModel)
             if (Url.IsLocalUrl(returnUrl))
                 return Redirect(returnUrl);
 
-            if (string.IsNullOrEmpty(returnUrl))
+            if (string.IsNullOrWhiteSpace(returnUrl))
```

Comment:
Following on from the comments above about the likelihood of `ReturnUrl == null` we may not need this check either.

Reply:
Yeah I agree.

Reply:
As above, I'm going to leave this as is as a malicious actor could replay the anti-forgery tokens.